.github/workflows/linters.yml

```diff
@@ -1 +1,71 @@
+name: Linters
+
+on: pull_request
+
+env:
+  FORCE_COLOR: 1
```

Comment:
I can see that tests are not running in the repository. I checked the files, all seems good, seems like github's check is just down. Try to leave comments in future similar situation to make it clear that you worked on the issue 😃